Encode.xs

```diff
@@ -417,6 +443,8 @@ process_utf8(pTHX_ SV* dst, U8* s, U8* e, SV *check_sv,
     int check;
     U8 *d;
     STRLEN dlen;
+    char esc[80]; /* need to store UTF8SKIP * 6 + 1 */
```

Comment:
Looks like `UTF8_MAXBYTES` is not available in older perl versions (which are supported by Encode). But I can add needed #define.